_meta/beat.yml

```diff
@@ -114,6 +114,25 @@ apm-server:
     # Set to false to disable the metrics endpoint
     #enabled: true
 
+  # Pipeline definitions are registered at the configured Elasticsearch instances.
+  # Registering pipelines therefore requires to have `output.elasticsearch` enabled and configured.
+  #
+  # If registering pipelines is disabled, you can still configure `output.elasticsearch.pipelines`
```

Comment:
probably because i never looked into pipelines, this comment is hard to understand for me: is the case that `output.elasticsearch.pipelines` is the same as `register.ingest.pipeline` for existing documents? or why do you mention it here?